account_group/README.rst

```diff
@@ -0,0 +1,112 @@
+===================
+Groups for accounts
+===================
+
+.. !!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!
+   !! This file is generated by oca-gen-addon-readme !!
+   !! changes will be overwritten.                   !!
+   !!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!
+
+.. |badge1| image:: https://img.shields.io/badge/maturity-Beta-yellow.png
+    :target: https://odoo-community.org/page/development-status
+    :alt: Beta
+.. |badge2| image:: https://img.shields.io/badge/licence-AGPL--3-blue.png
+    :target: http://www.gnu.org/licenses/agpl-3.0-standalone.html
+    :alt: License: AGPL-3
+.. |badge3| image:: https://img.shields.io/badge/github-OCA%2Faccount--financial--tools-lightgray.png?logo=github
+    :target: https://github.com/OCA/account-financial-tools/tree/10.0/account_group
+    :alt: OCA/account-financial-tools
+.. |badge4| image:: https://img.shields.io/badge/weblate-Translate%20me-F47D42.png
+    :target: https://translation.odoo-community.org/projects/account-financial-tools-10-0/account-financial-tools-10-0-account_group
+    :alt: Translate me on Weblate
+.. |badge5| image:: https://img.shields.io/badge/runbot-Try%20me-875A7B.png
+    :target: https://runbot.odoo-community.org/runbot/92/10.0
+    :alt: Try me on Runbot
+
+|badge1| |badge2| |badge3| |badge4| |badge5| 
+
+This modules brings to version 10 the account group feature from v11.
+
+**Table of contents**
+
+.. contents::
+   :local:
+
+Installation
+============
+
+If you have already the chart of accounts loaded on your company, you will
+need to update it through the module `account_chart_update`. Follow the
+instructions on that module for that.
+
+Configuration
+=============
+
+To configure account groups, you need to:
+
+#. Be "Account / Adviser" role.
+#. Go to *Invoicing > Configuration > Accounts Groups*.
+#. Create or modify existing groups.
+
+For assigning groups to account templates, you have to:
+
+#. Set the group on your account chart module or extension.
+#. Or develop/create UI access.
+
+When you have groups on your account templates, you can load a chart template
+for a new company, and they will be transferred to created accounts.
+
+Usage
+=====
+
+For assigning groups to accounts:
+
+#. Go to *Invoicing > Adviser > Chart of Accounts*.
+#. Edit one account and set "Group" field.
```

Comment:
I've tested as Usage said. But I guess other module will come depending on this one. Isn't it?